src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -77,6 +76,10 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
       toast.error('Please Select a Project Manager');
       return;
     }
+    if (!confirmDetails) {
```

Comment:
None of these are actually needed because of the yup validation

Reply:
good point ... I went ahead and deleted the toast confirmation for "confirmDetails". However, there is no yup validation for leadId and managerId. So, if I were to delete those two toast errors, no error would be presented to the user if they tried to submit the form without a lead or manager. Should I leave those ones as they are?

Reply:
oh wait, actually when I remove the confirmDetails toast check, I'm able to submit the form without clicking either button. Do you happen to know how to fix this?

Reply:
Just add it to the yup validation